pkgs/development/compilers/mint/crystal2nix.cr

```diff
@@ -0,0 +1,33 @@
+require "yaml"
```

Comment:
Should this tool really live in a mint-specific directory?

Reply:
That's a good question, and eventually I want to make it a standalone tool, but until I package more Crystal applications, I think it can live here for a bit.
I can remove it if you don't want to have it here, just thought it makes the work for future packages a bit easier.